src/onnx/onnx.cpp

```diff
@@ -52,6 +53,8 @@ namespace onnx = onnx_for_migraphx;
 
 struct onnx_parser
 {
+    std::string filename;
+    std::string path;
```

Comment:
is there a default value for the filename and path like the onnx file name appending something?

Reply:
I assign it when I call `get_abs_path`, but I can change it to be "." by default. The assignment will change once PR #664 is merged in.